drivers/ovsdbDriver.go

```diff
@@ -337,6 +352,27 @@ func (d *OvsdbDriver) CreatePort(intfName, intfType, id string, tag int) error {
 	return d.performOvsdbOps(operations)
 }
 
+//UpdatePolicingRate will update the ingress policing rate in interface table.
+func (d *OvsdbDriver) UpdatePolicingRate(intfName string, burst int, bandwidth int64) error {
+	log.Infof("Received ingress_policing_rate update")
+	bw := int(bandwidth)
+	intf := make(map[string]interface{})
+	intf["ingress_policing_rate"] = bw
```

Comment:
Please add some error checking here. If bandwidth is zero, we should return early.
If burst size is zero, we should not set it.